purchase_requisition_from_rfq/hooks.py

```diff
@@ -0,0 +1,14 @@
+# Copyright 2019 Ecosoft Co., Ltd (http://ecosoft.co.th/)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html)
+
+from odoo.addons.purchase_requisition.models.purchase import PurchaseOrder
+
+
+def post_load_hook():
```

Comment:
@ps-tubtim, I saw this function ? I didn't sure what is this for ? please tell me ...

Reply:
Deleted.